src/main/resources/org/nest/sympy/integration_test.py

```diff
@@ -0,0 +1,42 @@
+from sympy import *
+import misc
+import ode_to_prop_matrix as otpm
+import unittest
+
+class TestShapesConversionAndPropagatorMatrxiComputation(unittest.TestCase):
+    def test_shape_conversion(self):
+        shape_alpha = misc.ShapeFunction("shape_alpha", "e / tau * t * exp(-t / tau)")
+        self.assertEqual(2, shape_alpha.order)
+
+        shape_exp = misc.ShapeFunction("shape_exp", "exp(-t/tau)")
+        self.assertEqual(1, shape_exp.order)
+
+        shape_1 = misc.ShapeFunction("shape_1",  "e / tau * t * t * exp(-t / tau)")
+        self.assertEqual(3, shape_1.order)
+        # shape_2 has order 4
+        # shape_2 = misc.ShapeFunction("shape_2",  "e / tau * t * t *t * exp(-t / tau)")
+
+        shape_3 = misc.ShapeFunction("shape_3", "exp(-t/tau) + e / tau * t * exp(-t / tau)")
+        self.assertEqual(2, shape_3.order)
+
+        shape_4 = misc.ShapeFunction("shape_4", "cos(t)")
+        self.assertEqual(2, shape_4.order)
+
+        shape_5 = misc.ShapeFunction("shape_5", "sin(t)")
+        self.assertEqual(2, shape_5.order)
+
+    def test_propagator_matrix(self):
+        shape_alpha = misc.ShapeFunction("shape_alpha", "e / tau * t * exp(-t / tau)")
+        shape_exp = misc.ShapeFunction("shape_exp", "exp(-t/tau)")
+        shape_sin = misc.ShapeFunction("shape_sinb", "sin(t)")
+        shapes = [shape_alpha, shape_exp, shape_sin]
+
+        ode_var = "V_m"
+        ode_rhs = "-1/Tau * V_m-1/C * (shape_alpha + shape_exp + shape_sin + currents + I_E)"
+        prop_matrices, const_term, step_const = otpm.ode_to_prop_matrices(shapes, ode_var, ode_rhs)
+
+        pm, ps = otpm.prop_matrix_to_prop_step(prop_matrices, const_term, step_const, shapes, ode_var)
+        # checks that the computation doesn't fail!
```

Comment:
Which code does this comment refer to? Should it go above the code line above it?